src/components/App/index.tsx

```diff
@@ -273,7 +265,7 @@ export default class App extends Component<Props, State> {
 
     let bmp;
     try {
-      bmp = await createImageBitmap(file);
+      bmp = await decodeImage(file);
```

Comment:
@surma I think this should happen, right?

Reply:
+1